solr/solr-ref-guide/src/collection-management.adoc

```diff
@@ -350,31 +506,46 @@ update commands, which are directed now to `collection2`.
 [[delete]]
 == DELETE: Delete a Collection
 
-`/admin/collections?action=DELETE&name=_collection_`
+The DELETE action is used to delete a collection.
+
+[.dynamic-tabs]
+--
+[example.tab-pane#v1deletecollection]
+====
+[.tab-label]*V1 API*
+
+With the v1 API, the `delete` command must be capitalized as `DELETE`:
+
+[source,bash]
+----
+http://localhost:8983/solr/admin/collections?action=DELETE&name=newCollection
```

Comment:
> name=newCollection

This is a real nitpick, there's not much danger of confusion here idt.  But the collection name `newCollection` might confuse readers, as this API call has nothing to do with creating new collections.

Reply:
humm...  I was starting with  the idea that we created a "newCollection" at the very beginning.    how about "testCollection" throughout?

Reply:
Okay, went with `techproducts_v2` as the example!